src/app/euro.pipe.ts

```diff
@@ -0,0 +1,18 @@
+import { Pipe, PipeTransform } from '@angular/core';
+import { DecimalPipe } from '@angular/common';
+
+@Pipe({name: 'euro'})
+export class EuroPipe extends DecimalPipe implements PipeTransform {
```

Comment:
why not to use just [CurrencyPipe](https://angular.io/docs/ts/latest/api/common/index/CurrencyPipe-pipe.html)?